ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -32,18 +32,18 @@ using std::shared_ptr;
 using std::make_unique;
 
 // C
-#include <cmath>  // for M_PI and fabs
+#include <math.h>  // for M_PI, fabs, fmod
```

Comment:
Why switch to `math.h`? The preferred library to use here would be `cmath`.

Reply:
I read just the opposite in an upvoted StackOverflow article but don't want to argue such a small point -- changed it back to cmath.

Reply:
Interesting... I have only heard arguments for using only cmath in C++ code. What I have read says that the c* headers are like of the original C libraries adapted for C++ code, and their *.h counterparts should be considered deprecated. Granted, I think the only way using *.h over c* could cause trouble is if people are not careful with their type conversions, in particular when using intrinsic types that did not exist in C (e.g. https://cplusplus.com/forum/general/5207/).

Reply:
The argument I read is that they are equivalent, except that math.h guarantees definition in the global namespace (with possible definition in std), while cmath guarantees definition in std (with possible definition in global).  I wanted to use the functions without a namespace prefix, so math.h seemed more portable for this choice.  But portability is not much of an issue for OceanWATERS, and this issue is too small to ponder too much :)